cmd/release-controller-api/http.go

```diff
@@ -298,6 +298,7 @@ func (c *Controller) userInterfaceHandler() http.Handler {
 	mux.HandleFunc("/releasetag/{tag}", c.httpReleaseInfo)
 
 	mux.HandleFunc("/releasestream/{release}/release/{tag}", c.httpReleaseInfo)
+	mux.HandleFunc("/inconsistency/{release}/release/{tag}", c.httpInconsistencyInfo)
```

Comment:
Do we actually need an endpoint for this work?  I thought it was just a visual indicator on the main page.

Reply:
Viewing the inconsistency message is one of the requirements, but we do not necessarily need an endpoint for it.
The initial idea was to provide the data on the inconsistency when hovering over the warning icon. This might prove difficult in the current configuration since the body of the inconsistency message might be large (hundreds of entries).

Reply:
Yeah, that seems appropriate.  So, I would probably anchor this page here instead:
`mux.HandleFunc("/releasetag/{tag}/inconsistency", c.httpInconsistencyInfo) `

Reply:
Changed the location to /releasetag/{tag}/inconsistency.

Reply:
I know that I recommended changing this, but after my other requests, we might need to have this rooted where it was before.  Why?  Because we can get everything we need from the stream itself and not need to modify any other data structures...
So, `httpInconsistencyInfo` could become something like:
```
func (c *Controller) httpInconsistencyInfo(w http.ResponseWriter, req *http.Request) {
	// TODO implement the API for the inconsistency data
	start := time.Now()
	defer func() { klog.V(4).Infof("rendered in %s", time.Now().Sub(start)) }()

	tagInfo, err := c.getReleaseTagInfo(req)
	if err != nil {
		http.Error(w, err.Error(), http.StatusNotFound)
		return
	}
	
	for _, tag := range tagInfo.Info.Release.Source.Spec.Tags {
		if value, ok := tag.Annotations[releasecontroller.ReleaseAnnotationInconsistency]; ok {
			klog.V(4).Infof("Found: %s", value)
		}
	}
}
```

Reply:
No worries! I have changed it. At the current implementation, the endpoint returns a JSON only.

Reply:
The endpoint returns an HTML page now.
![Screenshot from 2022-03-02 22-37-42](https://user-images.githubusercontent.com/77680331/156453877-1ccdefb6-7d67-4aaf-9e05-1df5b924465c.png)